### ISSUE TITLE
JAWS remains silent when caret moves within a textbox

### ISSUE THREAD

**Issue by MRaybuck** [FreedomScientific/VFO-standards-support#62]
### Summary

JAWS does not function with the UIA Document provider sample from Microsoft. With a couple alterations, Windows Narrator will announce what letter the caret is adjacent to as it moves through the document's text. It will also read the text aloud as the mouse cursor moves over it. JAWS remains silent when mousing over text and will only announce "Blank." as the caret moves along a text line instead of which character it is adjacent to.

To reproduce this behavior simply do the following:
1) Build and run the sample provided below.
2) Click anywhere to put the cursor on some line of text.
3) Turn on Narrator and navigate around the sample using the Arrow Keys.
4) Turn off Narrator and turn on JAWS. Use the Arrow Keys to navigate around.

### Expected result

JAWS should announce what character the caret is adjacent to as it moves along a line of text.

### Actual result

JAWS will repeatedly announce "Blank." as the caret moves alone a line of text.

### Example
The sample mentioned can be downloaded from https://github.com/MRaybuck /UIADocumentProviderSample

Build and run the sample. It may require Visual Studio 2013 in order to build.

#### JAWS version and build number
JAWS 2018.1803.24 64-bit

 
#### Operating System and version
Windows 10 Education Version 1709 OS Build 16299.309

**Comment by stevefaulkner** [comment]
@MRaybuck This is for a desktop app?

**Comment by MRaybuck** [comment]
Yes, this is for a desktop application. I put a sample VS2013 project that reproduces the behavior on my GitHub account. https://github.com/MRaybuck/UIADocumentProviderSample

**Comment by jducrot** [comment]
@MRaybuck We're getting caught up on triaging this issue but it appears your original link is not valid anymore. If you still have access to the code, would you mind sharing it again?
